### PR TITLE
Support positional-only parameters

### LINKED ISSUE #35: Add support for positional-only parameters
Parameters such as https://github.com/python/typeshed/blob/808a431a57e7efb2ec8afa97840c368e760d2e11/stdlib/collections/__init__.pyi#L52 should have had defaults added to them by now, but haven't. I think the reason is that there isn't a `__dict` parameter at runtime (there's a positional-only `dict` parameter: https://github.com/python/cpython/blob/666c0840dcac9941fa41ec619fef8d45cd849a0b/Lib/collections/__init__.py#L1112), causing `KeyError` to be raised here, which causes stubdefaulter to just ignore the parameter and move on:

https://github.com/JelleZijlstra/stubdefaulter/blob/719c3b52eafe9a726a7e666692821bf4c809ac29/stubdefaulter.py#L74

### PR DESCRIPTION
**Pull request by AlexWaygood**
Fixes #35

### CODE REVIEW

**Comment by AlexWaygood** [comment]
*Shakes fist at Python 3.7*

File: test_stubdefaulter.py [hunk 1]
```diff
@@ -48,10 +49,15 @@ class FooEnum(str, enum.Enum):
 
 def strenum_default(x=FooEnum.FOO):
     return str(x)
+
+if sys.version_info >= (3, 8):
+    def pos_only(x=5, /):
```
Comment (JelleZijlstra) [inline test_stubdefaulter.py:54]:
```suggestion
    def pos_only(x=5):
```
Seems like the easiest solution. Nothing is verifying that the stub used in the test is precisely correct.

Comment (AlexWaygood) [reply test_stubdefaulter.py:54]: I guess...!

Comment (JelleZijlstra) [reply test_stubdefaulter.py]: You'll also have to remove the version check or stubdefaulter won't add the default when run under 3.7. (Obviously there's other ways to fix if you prefer.)

**Comment by AlexWaygood** [comment]
where is the SyntaxError??

File: test_stubdefaulter.py (hunk 1 above)
Comment (AlexWaygood) [reply test_stubdefaulter.py]: oh, of course

**Review by JelleZijlstra — APPROVED** [review]

**Pull request merged**